src/include/migraphx/register_op.hpp

```diff
@@ -33,15 +33,33 @@
 namespace migraphx {
 inline namespace MIGRAPHX_INLINE_NS {
 
+// unregister all ops for specified target, useful when unloading dynamically plugged-in target lib
+void unregister_op(const std::string& op_name);
```

Comment:
This should take an `operation` to be consistent.

Reply:
maps key is `op_name` that's why it is std::string.